File: src/main/kotlin/demo/kotliln/boardgame/service/rating/RateRepository.java
package demo.kotliln.boardgame.service.rating;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface RateRepository extends CrudRepository<RateEntity, Long> {

    List<RateEntity> findByBoardGameName(String boardGameName);
}

File: src/main/kotlin/demo/kotliln/boardgame/service/rating/repositories/RateRepository.kt
package demo.kotliln.boardgame.service.rating.repositories

import demo.kotliln.boardgame.service.rating.entities.RateEntity
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface RateRepository : CrudRepository<RateEntity, Long> {
    fun findByBoardGameName(boardGameName: String): List<RateEntity>
}